libbeat/monitoring/report/elasticsearch/client.go

```diff
@@ -77,6 +80,8 @@ func (c *publishClient) Connect() error {
 	}
 
 	debugf("XPack monitoring is enabled")
+	debugf("Publishing to %v cluster", c.params["_format"])
```

Comment:
Do we know for sure `_format` is set? Want to make sure that because of log nothing is panicing.

Reply:
It should be set for sure at this point, but that also requires a check in the constructor. I had this at one point but alas it seems to have got lost in a rebase 😢. Will re-add.